tests/kernel/fpu_sharing/generic/src/test_common.h

```diff
@@ -9,7 +9,7 @@
 /*
  * Test Thread Parameters
  */
-#define THREAD_STACK_SIZE	1024
+#define THREAD_STACK_SIZE	(1024 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
Seems we have both `ZTEST_STACKSIZE` and `TEST_EXTRA_STACKSIZE`.

Should this be a `(ZTEST_STACKSIZE + TEST_EXTRA_STACKSIZE)` ?
and should we consider to align those two settings ?
@galak @nashif any thoughts on this ?

Reply:
feels like aligning the two and phasing one out if possible.

Reply:
ZTEST_STACKSIZE is not related here, this is for the threads that are created within the test, ZTEST_STACKSIZE is for ztest itself.

Reply:
Thank you @tejlmand for the review. `(1024 + CONFIG_TEST_EXTRA_STACKSIZE)` is intentional.

Reply:
thanks for clarification.